api/managementpb/backup/common.proto

```diff
@@ -18,3 +21,25 @@ enum BackupMode {
   INCREMENTAL = 2;
   PITR = 3;
 }
+
+// File represents file or folder on a storage.
+message File {
+  string name = 1 [(validate.rules).string.min_len = 1];
+  bool is_directory = 2;
```

Comment:
Minor, but we mixing `directory` and `folder` in one API

Reply:
Folder is a container while directory is a hierarchical structure. To me doesn't look like mixing, rather two different words for two different meaning.